### ISSUE TITLE
Ensure it is trivial to do GSEM with umxRAM

### ISSUE THREAD

**Issue by tbates** [tbates/umx#146]
Consider wrappers for the GWAS pipeline also.
https://github.com/MichelNivard/GenomicSEM/wiki

**Comment by mcneale** [comment]
Yes, there are some weirdnesses with Lavaan or OpenMx WLS(MV) implementations that make them disagree.  Others have reported inconsistencies between Lavaan and MPlus.  At least we can triangulate a bit - and with two of them being open source we can actually figure out WHY they differ if they do.

For more complicated modeling, OpenMx will be faster than Lavaan due to using C++ instead of R for computation & optimization.  However, for WLS, where the weight matrix has to be inverted only once, most of the heavy lifting has already been done, with WLS model-fitting proceeding very rapidly.

**Comment by tbates** [comment]
done
